src/codeu/chat/common/NetworkCode.java

```diff
@@ -36,4 +36,7 @@
       RELAY_READ_RESPONSE = 28,
       RELAY_WRITE_REQUEST = 29,
       RELAY_WRITE_RESPONSE = 30;
+     public static final SERVER_INFO_REQUEST = 31;
+     public static final SERVER_INFO_RESPONSE = 32;
```

Comment:
Add this in the same way as the other codes above, i.e. `SERVER_INFO_REQUEST = 31,`. No need to have duplicate `public static final` statements here.